src/config/app.ts

```diff
@@ -1,8 +1,13 @@
+// NOTE(DP): use instead of ../../-like imports path-aliases
+//           see more:
+//           https://dev.to/larswaechter/path-aliases-with-typescript-in-nodejs-4353
 import { loadEnv } from '../helpers/load-env.js'
 import { validateEnv } from '../helpers/validate-env.js'
 import { startBot } from './bot.js'
 import { connectToDb } from './database.js'
 
+// NOTE(DP): looks like startApp() is not related to src/config/ scope
```

Comment:
Initialization of app, the same concept as the rest of files there. Probably `config` should be renamed